uint8[] camera_0  # 图像数据
uint8[] camera_1  # 图像数据
uint8[] camera_2  # 图像数据
uint8[] camera_3  # 图像数据
float64[] robot_joint  # 机器人关节角
